shared/people/invite-friends/index.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as Styles from '../../styles'
+import * as Container from '../../util/container'
+import {usePhoneNumberList} from '../../teams/common'
+
+const InviteFriends = () => {
+  const dispatch = Container.useDispatch()
+  const nav = Container.useSafeNavigation()
+  const onClose = () => dispatch(nav.safeNavigateUpPayload())
+  const defaultCountry = Container.useSelector(s => s.settings.phoneNumbers.defaultCountry)
+
+  const [emails, setEmails] = React.useState('')
+  const {phoneNumbers, setPhoneNumber, addPhoneNumber, removePhoneNumber} = usePhoneNumberList()
+
+  // disabled if both are empty or if there are some invalid phone numbers
+  const disabled =
+    (!emails && phoneNumbers.every(pn => !pn.phoneNumber)) ||
+    phoneNumbers.some(pn => pn.phoneNumber && !pn.valid)
+
+  return (
+    <Kb.Modal
+      mode="DefaultFullHeight"
+      onClose={onClose}
+      header={{title: Styles.isMobile ? 'Invite friends' : 'Invite your friends to Keybase'}}
+      footer={{
+        content: (
+          <Kb.Box2 direction="vertical" gap="medium" fullWidth={true}>
+            <Kb.Button fullWidth={true} label="Send invite" disabled={disabled} />
+            {!Styles.isMobile && (
+              <Kb.Box2 direction="vertical" gap="tiny" fullWidth={true}>
+                <Kb.Text type="BodySmall" center={true}>
+                  or share a link:
+                </Kb.Text>
+                <Kb.CopyText textType="BodySemibold" text="https://keybase.io/download" />
+              </Kb.Box2>
+            )}
+          </Kb.Box2>
+        ),
+      }}
+    >
+      <Kb.Box2 direction="vertical" gap="small" style={styles.container}>
+        <Kb.Icon type="icon-illustration-invite-friends-460-96" style={{width: '100%'}} />
+        <Kb.Box2 direction="vertical" gap="small" fullWidth={true} style={styles.content}>
+          <Kb.Box2 direction="vertical" gap={Styles.isMobile ? 'xtiny' : 'tiny'} fullWidth={true}>
+            <Kb.Text type="BodySmallSemibold">By email address (separate with commas)</Kb.Text>
+            <Kb.LabeledInput
```

Comment:
something is fishy here
<img width="449" alt="image" src="https://user-images.githubusercontent.com/4698687/77083806-72e7d280-69d4-11ea-8db2-77878fb8d62e.png">

Reply:
oy

Reply:
Ok it's an issue in `LabeledInput`, fixed and affected storyshots are alright.